pkg/ip/ip.go

```diff
@@ -159,6 +159,24 @@ func (i *Range) UnmarshalText(text []byte) error {
 	return nil
 }
 
+// ParseIPandMask parses a CIDR format address (e.g. 1.1.1.1/8)
+func ParseIPandMask(s string) (net.IPNet, error) {
```

Comment:
I tried to replace usage with `netlink.ParseIPNet`, but that gave me compile errors for the windows exe